lib/Model/SendSmtpEmailReplyTo.php

```diff
@@ -36,7 +36,7 @@
  * SendSmtpEmailReplyTo Class Doc Comment
  *
  * @category Class
- * @description Email (required), along with name (optional), on which transactional mail recipients will be able to reply back. For example, {&#39;email&#39;:&#39;ann6533@example.com&#39;, &#39;name&#39;:&#39;Ann&#39;}.
+ * @description Email (required), along with name (optional), on which transactional mail recipients will be able to reply back. For example, {\&quot;email\&quot;:\&quot;ann6533@example.com\&quot;, \&quot;name\&quot;:\&quot;Ann\&quot;}.
```

Comment:
`\&quot;` needs to be replaced

Reply:
updated

Reply:
[](https://github.com/sendinblue/APIv3-php-library/commit/0b86513957d22c3fcd0598f5c961be2513f6c8d6)> `\&quot;` needs to be replaced

Reply:
> [](https://github.com/sendinblue/APIv3-php-library/commit/0b86513957d22c3fcd0598f5c961be2513f6c8d6)> `\&quot;` needs to be replaced
> 
>